src/utils/getUniqueIdGenerator.spec.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-env mocha */
+import {assert} from 'chai';
+import getUniqueIdGenerator from './getUniqueIdGenerator';
+
+describe('utils/getUniqueIdGenerator', () => {
+  let gen;
+  let anotherGen;
+
+  beforeEach(() => {
+    gen = getUniqueIdGenerator();
+    anotherGen = getUniqueIdGenerator();
+  });
+
+  describe('one generator', () => {
+    it('should generate different uniqueIds each time it is invoked', () => {
+      assert.notEqual(gen(), gen());
```

Comment:
@oliviertassinari I agree with you. 😲Using `notEqual` focus on the the generator's use cases, not its function. I should add some tests to `TextField`.